src/F2_autoreport.py

```diff
@@ -527,8 +529,29 @@ def create_app(results_json):
                             make_dash_data_table(df_simsettings, "Simulation Settings"),
                         ],
                     ),
-                    insert_subsection("Energy demand", "content"),
-                    insert_subsection("Energy system components", "content"),
+                    insert_subsection(title="Energy demand",
+                                      content=[
+                                          html.Div(
+                                              children=[
+                                                  insert_body_text(
+                                                      "The simulation was performed for the energy system "
+                                                      "covering the following sectors: "
+                                                  ),
+                                                  insert_body_text(f"{sec_list}"),
```

Comment:
Why not inserted in the body_text above?

Reply:
It displays it Electricity in the same line. I tried multiline strings, but somehow it did not work. Will look into it again.